aspnetcore/tutorials/signalr-blazor-webassembly.md

```diff
@@ -5,16 +5,14 @@ description: Create a chat app that uses ASP.NET Core SignalR with Blazor WebAss
 monikerRange: '>= aspnetcore-3.1'
 ms.author: riande
 ms.custom: mvc
-ms.date: 05/11/2020
+ms.date: 05/19/2020
 no-loc: [Blazor, "Identity", "Let's Encrypt", Razor, SignalR]
 uid: tutorials/signalr-blazor-webassembly
 ---
 # Use ASP.NET Core SignalR with Blazor WebAssembly
```

Comment:
I think the prereqs listed in this topic need to be updated. They suggest VS 16.4 instead of latest.

Reply:
They come in via INCLUDES files for 3.1. I guess the doc author who set them up hard-coded that. I'll see if I can fix it (at least for this topic).

Reply:
They put "or later" as they're supposed to. This is the normal way for docs until we hit the next SDK release.

Reply:
The prereq for VS says to use 16.4 or later, but then in the "Create a hosted Blazor WebAssembly app project" section we say that 16.6 or later is required, which feels a bit inconsistent.

Reply:
I see. I'll replace the INCLUDE and resolve it manually then.

I certainly can't change the INCLUDE, as it's wired into a lot of docs.